Database/src/main/resources/changelog1.xml

```diff
@@ -838,4 +838,25 @@
         </createTable>
         <addPrimaryKey tableName="tbl_tenant_config" columnNames="tenant_id, key" constraintName="tenant_specific_key" />
     </changeSet>
+    
+    <changeSet author="CAF" id="adding_repository_table">
+        <createTable tableName="tbl_repository_config">
+            <column name="tenant_id" type="VARCHAR(40)">
+                <constraints nullable="false"/>
+            </column>            
+            <column name="repository_id" type="VARCHAR(40)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="key" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="value" type="CLOB">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addPrimaryKey tableName="tbl_repository_config" columnNames="tenant_id, repository_id, key" 
+                       constraintName="repository_specific_key" />
+        <addForeignKeyConstraint baseTableName="tbl_repository_config" baseColumnNames="tenant_id, key" constraintName="fk_tenant_key" 
```

Comment:
Fix the formatting here, I can't review properly as not familiar with Liquibase. Sorry.